lib/core/common/helpers/validation_helper.dart

```diff
@@ -0,0 +1,17 @@
+class ValidationHelper {
+  static final RegExp _emailRegExp = RegExp(
+    r'^[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$',
+  );
+  static final RegExp _passwordRegExp = RegExp(
```

Comment:
_passwordRegExp -> passwordRegExp. if using static variable do not named that as a private